.babelrc

```diff
@@ -1,13 +1,14 @@
 {
-    "presets": [
-        [
-            "@babel/preset-env",
-            {
-                "targets": {
-                    "node": "current"
-                }
-            }
-        ],
-        "next/babel"
-    ]
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "node": "current"
+        }
+      }
+    ],
+    ["@babel/preset-react", { "runtime": "automatic" }],
```

Comment:
Do you really need to add these? As attached link from repo maintainer, this isn't required after `next/babel`

Reply:
I talked with Abdul and he said it is good to go, since it won't conflict with anything.